templates.json

```diff
@@ -696,7 +716,8 @@
         "value": "typescript",
         "path": "discount-details-function-settings-block"
       }
-    ]
+    ],
+    "deprecatedFromCliVersion": "3.85.1"
```

Comment:
We're using the upcoming patch which will be available on Oct 1st when the preact template will be the only available flavor.